packages/react/src/River/River.fixtures.stories.tsx

```diff
@@ -120,6 +120,86 @@ export const AlternativeHeadingSize: StoryFn<typeof River> = () => (
   </Container>
 )
 
+const ExampleTrailingComponentOne = () => (
```

Comment:
This doesn't use a `River` component. Is that intentional?

Reply:
Yeah, this is a local story fixture that we're using for the River's `trailingComponent`. It's used in the adjacent stories, and not a story itself (not exported).